libs/wingc/src/jsify.rs

```diff
@@ -1041,6 +963,15 @@ impl<'a> JSifier<'a> {
 		for (method_name, method_refs) in refs {
 			bind_method.open(format!("if (ops.includes(\"{method_name}\")) {{"));
 			for (field, ops) in method_refs {
+				// field might be a dotted accessor, like foo.bar.baz, in which case
+				// we first bind to `foo` as an op
+				if field.split(".").count() > 1 && field.split(".").next().unwrap() != "this" {
+					let first_part = field.split(".").next().unwrap();
```

Comment:
performance nit: we should call `field.split(".").next()` only once before the `if` (we call it 3 times here). Also consider using `splitn(1,".")` instead of `split`.

Reply:
Good catch. also TIL about `splitn` 😮